fundingmanager.go

```diff
@@ -1190,6 +1199,13 @@ func (f *fundingManager) waitForFundingConfirmation(completeChan *channeldb.Open
 		return
 	}
 
+	// In case the fundingManager is stopped at some point during the
+	// remaining part of the opening process, we must wait for this process
+	// to finish (either successully or with some error), before the
+	// fundingManager can be shut down.
+	f.wg.Add(1)
```

Comment:
This wait group increment should be lifted to the point in which the goroutine is actually launched.